tests/framework/framework.go

```diff
@@ -73,6 +82,12 @@ func init() {
 	master = flag.String("master", "", "master url:port")
 }
 
+func (c *FrameworkConfig) NewFramework(prefix string) *Framework {
```

Comment:
I do like this approach but just left a comment that I think David's idea of a separate options struct parm is more often seen in k8s and works fine for our needs. My comments tonight reflect that approach...